### ISSUE TITLE
Add `parking:lane:*=no` to the edit menu

### ISSUE THREAD

**Issue by tordans** [zlant/parking-lanes#34]
We use `parking:lane:*=no` to show that there is no parking but also no special sign that would justify a `no_parking` tag – especially when a road is mapped in two separate ways since there is a barrier in between.

I suggest adding it to the edit mode.
I think the view mode is already good, since this mapping is – as far as I know – mostly used on separately mapped roads so ignoring it in the view mode already looks good.

Example: 
- https://www.openstreetmap.org/way/764644564
- https://zlant.github.io/parking-lanes/#18/52.47765/13.43254

CC @SupaplexOSM – Does this ticket describe the usage of this value correclty?

**Comment by SupaplexOSM** [comment]
Yes, your example is a good one for the "no"-value, as I understand and use it. But it's use is rarely discussed and documented so I'm not sure how well it is supported in the community. Note: I think there is a consensus that a **"no_parking" does not necessarily have to be signed** – e.g. in a curve or in a narrow street. My understanding: But there should be at least a theoretical option to park there if the situation would be different (e.g. no curve or a wider road).

On the inner side of "dual carriageway" roads (OSM tag for that in rare use), which are not signed as one-way roads, parking is usually neither intended nor allowed. For this fine differentiation I use "parking:lane:left=no".

There might be other cases where this tag could be useful, but so far there has been very little discussion about it and the documentation is sparse.

**Comment by tordans** [comment]
Thanks @zlant I see 'no' is now part of the value list, that is great.

What do you think about adding it to the visualization as well as a color?

Right now, those `parking:lane` are presented "blank", but they could also be considered a variant of red, since there is no parking allowed.

| edit  | present |
| ------------- | ------------- |
| <img width="684" alt="Bildschirmfoto 2021-09-17 um 08 13 08" src="https://user-images.githubusercontent.com/111561/133733611-28991030-b443-4528-b795-4f26ef9f3501.png">  | <img width="733" alt="Bildschirmfoto 2021-09-17 um 08 13 15" src="https://user-images.githubusercontent.com/111561/133733619-ace66248-b82b-4a06-a60d-6a3a3d83d65e.png">  |
